src/pymor/operators/numpy.py

```diff
@@ -6,6 +6,8 @@
 
 This module provides the following |NumPy|-based |Operators|:
 
+- |NumpyHankelOperator| implicitly constructs a Hankel operator from a |NumPy array| of Markov
+  parameters.
```

Comment:
I think this should be moved to the bottom of the list as it is a rather special-interest operator.

Reply:
I agree. I assumed that this list was in reverse order to the order of classes.